tools/docker/daml-all-in-one/supervisord.conf

```diff
@@ -10,8 +10,17 @@ autostart=true
 autorestart=true
 stderr_logfile=/var/log/daml.err.log
 stdout_logfile=/var/log/daml.out.log
+priority=10
 
 
+[program:damlledger]
+# command=daml build && && daml sandbox --wall-clock-time --dar ./.daml/dist/quickstart-0.0.1.dar && daml json-api --config json-api-app.conf
```

Comment:
@raynatopedrajeta Please delete the commented code if no longer needed. If it's useful to keep it around because for some sort of testing you sometimes want to temporarily change this then I'd recommend opening a new section in the readme and putting your nodes there with explanations on how and what to test/debug with the specific commented code when in use.